src/main/java/com/google/api/codegen/transformer/csharp/CSharpSurfaceNamer.java

```diff
@@ -34,4 +37,25 @@ public String getStaticLangReturnTypeName(Method method, MethodConfig methodConf
     return getModelTypeFormatter().getFullNameFor(method.getOutputType());
   }
 
+  @Override
+  public String getApiWrapperClassName(Interface interfaze) {
+    return className(Name.upperCamel(interfaze.getSimpleName(), "Client"));
+  }
+
+  @Override
+  public String getCallableName(Method method) {
+    return "_" + varName(Name.upperCamel("Call", method.getSimpleName()));
```

Comment:
It's a convention in C# to prefix fields with an underscore.
But placing the underscore within the argument to Name.upperCamel causes it to complain that the argument isn't in upperCamel format.